vite.config.ts

```diff
@@ -72,24 +80,29 @@ export default defineConfig(({ mode }: UserConfig): UserConfig => {
       ],
       [...(mode === "production" ? [minifyHTML(), compression()] : [])],
       [...(process.env.TLS_DEV_SERVER === "1" ? [basicSsl()] : [])],
-      replicaForwardPlugin({
-        replicaOrigin: "127.0.0.1:4943",
-        forwardDomains: ["icp0.io", "ic0.app"],
-        forwardRules: [
-          {
-            hosts: ["nice-name.com"],
-            canisterName: "test_app",
-          },
-          ...(process.env.NO_HOT_RELOAD === "1"
-            ? [
-                {
-                  hosts: ["identity.ic0.app", "identity.internetcomputer.org"],
-                  canisterName: "internet_identity",
-                },
-              ]
-            : []),
-        ],
-      }),
+      {
+        ...replicaForwardPlugin({
+          forwardDomains: ["icp0.io", "ic0.app"],
+          forwardRules: [
+            {
+              hosts: ["nice-name.com"],
+              canisterName: "test_app",
+            },
+            ...(process.env.NO_HOT_RELOAD === "1"
+              ? [
+                  {
+                    hosts: [
+                      "identity.ic0.app",
+                      "identity.internetcomputer.org",
+                    ],
+                    canisterName: "internet_identity",
+                  },
+                ]
+              : []),
+          ],
+        }),
+        apply: () => isServe,
```

Comment:
What does this do?

Reply:
Is applies the plugin only when `true`: https://vitejs.dev/guide/api-plugin#conditional-application

Reply:
Ok, thanks!